packages/adapter-pg/README.md

```diff
@@ -1,3 +1,70 @@
 # @prisma/adapter-pg
 
-**INTERNAL PACKAGE, DO NOT USE**
+Prisma driver adapter for the [Postgres `pg` driver](https://github.com/brianc/node-postgres).
+
+> **Note:**: Support for the `pg` driver is available from Prisma versions [5.4.2](https://github.com/prisma/prisma/releases/tag/5.4.2) and later.
+
+The [Postgres `pg` driver](https://github.com/brianc/node-postgres) is a low-latency Postgres driver for JavaScript and TypeScript that allows you to query data any environment, including serverless and edge clouds.
+
+## Getting started
+
+To get started, enable the `driverAdapters` Preview feature in your Prisma schema:
+
+```prisma
+// schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+> **Note**: Make sure your connection string exists in your `.env` file.
+>
+> ```bash
+> DATABASE_URL="postgres://user:password@server.us-east-2.aws.com/yourdb"
+> ```
```

Comment:
This is the same `.env` advice we gave for Neon and PlanetScale, but are we sure we want to keep it this way? `.env` files do not work in all platforms.

For instance, they do not work on Cloudflare Workers. Rather, one could use:
- A `.dev.vars` file when developing with `wrangler dev`, following the same format and semantics as the `.env` snippet above
- A `[env.production.vars]` / `[env.staging.vars]` section in `wrangler.toml` for setting the variables for deployments.

I'm bringing this up so we can decide how we should tailor the README. Ideally, specifics about setting up env variables depending on the platform should live in Prisma Docs, and we should just add a link to that page in here.

Reply:
Agree! In my suggestion, I mentioned in the beginning that the `DATABASE_URL` needs to be set to the Postgres connection string without prescribin how to do it (but still mentioning `.env` as a a possibility).